app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -3,17 +3,20 @@
 module Idv
   module InPerson
     class ReadyToVerifyPresenter
+      include ActionView::Helpers::TagHelper
```

Comment:
Is this helper needed? I don't see any tag helpers being used in this class.

Reply:
No. Deleted with commit 6bbded93abd533ab2bfcbf0ef42a094aedf12a74